### PR TITLE
118 feature show collision box in editor

### PR DESCRIPTION
**Pull request by lucasdpt**
CollisionBox are now visible in editor

### CODE REVIEW

File: editor/src/Layers/Viewport/ShowUtils.hpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type [WSL: Ubuntu-22.04]
+** File description:
+** ShowUtils
+*/
+
+#ifndef SHOWUTILS_HPP_
```
Comment (Sinan-Karakaya) [inline editor/src/Layers/Viewport/ShowUtils.hpp:8]: use pragma once, not ifndef guards

**Review by Sinan-Karakaya — COMMENTED** [review]
Where is the toggle to enable/disable the visibility of the collision boxes?

File: editor/src/Layers/Viewport/ShowUtils.hpp (hunk 1 above)
Comment (lucasdpt) [reply editor/src/Layers/Viewport/ShowUtils.hpp:8]: I will do that

**Comment by lucasdpt** [comment]
> Where is the toggle to enable/disable the visibility of the collision boxes?

Add in last commit

**Review by Sinan-Karakaya — APPROVED** [review]

**Pull request merged**